collaborative_filtering/tests_similarity.py

```diff
@@ -0,0 +1,15 @@
+import itertools
+
+'''
+    Tests all elements of a matrix for a certain value. 
+    If activated the positions of the value will be printed on screen.
+    Returns True if the value is found in any position.
+'''
+def test_matrix_for_value(matrix, value, display_output_activated=False):
+    value_found = False
+    for element1, element2 in itertools.product(range(matrix.shape[0]), range(matrix.shape[1])):
+        if matrix[element1, element2] == value:
+            value_found = True
+            if display_output_activated:
+                print(str(value) + " at : ( " + str(element1) + " / " + str(element2) + " )")
+    return value_found
```

Comment:
I like to have the test files in a separate directory for a "tidier" project.
You might want to move this file to "test/collaborative_filtering/test_similarity.py".
The downside is, as far as I understood it, that you somehow will need to mess with the `sys.path`.
For me [this](https://github.com/lmaraite/Collaborative_Filtering/blob/42b34f057f547d6880982ec35f12f68343f3da0b/test/collaborative_filtering/__init__.py) worked.